src/sections/Hero/style.module.scss

```diff
@@ -0,0 +1,62 @@
+@use "src/styles/vars.scss" as vars;
+
+@import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
+
+.heroContainer {
+    background-image: linear-gradient(180deg, #100b44 15%, #a78590 98%);
+    min-height: 100vh;
+    font-family: Rubik, sans-serif;
+    padding-top: 10px;
+
+    .heroTitle{
+        position: absolute;
+        top: 5%;
+        z-index: 1;
+    }
+
+    .heroPlanet{
+        max-width: 60vh;
+        max-height: 60vh;
+        position: absolute;
+        top: 60%;
+        bottom: 0;
+        right: 0;
+        z-index: 1;
+    }
+
+   .heroDescription{
+        display: flex;
+        text-align: center;
+        justify-content: center;
+        flex-direction: column;  
+        position: absolute;
+        height: 100vh;
+        margin-top: 145px;
+        left: 50%;
+        margin-left: -275px;
+        z-index: 2;
+        
+        .heroSubtitle{
```

Comment:
![Screen Shot 2022-10-02 at 4 41 48 PM](https://user-images.githubusercontent.com/33165426/193481526-3a9d26df-4b19-4df7-ad39-df6172c4c16a.jpg)

Lots of small nitpicks on the responsive details but we don't want the text to overlap itself or be touching the edge.

Adding a line-height value to the text will space it out better vertically and we can probably just use margin-top to shift the content down without too many hardcoded values here.
![Screen Shot 2022-10-02 at 4 43 09 PM](https://user-images.githubusercontent.com/33165426/193481562-1090387e-eaf1-499e-835f-8c78d25416e3.jpg)